README.md

```diff
@@ -67,3 +67,57 @@ teracy-dev:
         branch: develop
       require_version: ">= 0.4.0-SNAPSHOT"
 ```
+
+## Supported configuration
+These are the supported configuration in teracy-dev-core: `variables`, `vagrant`, `default`, `node`.
+You can see the details about the default configuration in the `config.yaml` file.
+
+
+### Examples
+
+This section gives you some examples about the supported configuration in teracy-dev-core, so you should know how
+to override the default configuration.
+
+You should add your new configuration into the `workspace/teracy-dev-entry/config_override.yaml`.
+
+*vagrant*
+
+Please see an example at `workspace/teracy-dev-essential/blob/develop/config.yaml`.
+
+*default*
+
```

Comment:
need example config of `default`

Reply:
@hieptranquoc Please help me to give the example for it :)